test/uk/gov/hmrc/merchandiseinbaggage/pact/MibConnectorContractSpec.scala

```diff
@@ -153,14 +154,14 @@ class MibConnectorContractSpec extends BaseSpecWithApplication with CoreTestData
       interaction
         .description("check EoriNumber")
         .provided(s"checkEoriNumberTest")
-        .uponReceiving(GET, s"${checkEoriUrl}GB123")
+        .uponReceiving(GET, s"${config.mibCheckEoriUrl}GB123")
```

Comment:
Can just be `.uponReceiving(GET, "/declare-commercial-goods/validate/eori/GB123")`